app/forms/steps/outgoings/board_and_lodging_form.rb

```diff
@@ -41,10 +42,14 @@ def persist!
         outgoings_payment = crime_application.outgoings_payments.find_by(
           payment_type: OutgoingsPaymentType::BOARD_AND_LODGING.to_s
         )
+
         outgoings_payment.update!(
           amount: amount,
           frequency: frequency,
-          details: attributes.slice('board_amount', 'food_amount', 'payee_name', 'payee_relationship_to_client')
+          payee_name: payee_name,
+          payee_relationship_to_client: payee_relationship_to_client,
+          board_amount: board_amount.to_i, # Force explicit cast to conform to Schema
+          food_amount: food_amount.to_i,
```

Comment:
@timpeat note I am forcing explicit casting of a `pence` type which is actually a virtual attribute (e.g. `board_amount` is actually nested inside the outgoingspayment.metadata jsonb column, but is defined as a virtual attribute so it can be called like a normal attribute).